README.rst

```diff
@@ -46,7 +46,9 @@ Usage Example
     # Configure I2C mode
 
     iss = UsbIss()
-    iss.open("COM3")
+    p = iss.get_port(vid=0x04D8, pid=0xFFEE)
+    iss.open(p)
+    # iss.open("COM3")
```

Comment:
A good simplification to the open function would be to replace all of this with `iss.open()` and have it do the right thing.